base/linalg/cholesky.jl

```diff
@@ -94,7 +94,8 @@ function chol!(x::Number, uplo)
     rxr = sqrt(rx)
     convert(promote_type(typeof(x), typeof(rxr)), rxr)
 end
-chol(x::Number, uplo::Symbol=:U) = chol!(x, uplo)
```

Comment:
Yes, I cannot really think of something which deserves the name number where this would be a problem.